README.md

```diff
@@ -20,56 +20,54 @@ Download publish settings file from https://manage.windowsazure.com/publishsetti
 package main
 
 import (
-    "fmt"
+	"encoding/base64"
+	"fmt"
 
-    "github.com/Azure/azure-sdk-for-go/management"
-    "github.com/Azure/azure-sdk-for-go/management/hostedservice"
-    "github.com/Azure/azure-sdk-for-go/management/virtualmachine"
-    "github.com/Azure/azure-sdk-for-go/management/vmutils"
```

Comment:
I'm not sure what u mean. I believe Example() method is the copy of the readme. So I updated both to have working examples.